pkgs/by-name/um/umu-launcher/package.nix

```diff
@@ -2,13 +2,16 @@
   buildFHSEnv,
   lib,
   umu-launcher-unwrapped,
+  extraPkgs ? pkgs: [ ],
+  extraLibraries ? pkgs: [ ],
 }:
 buildFHSEnv {
   pname = "umu-launcher";
   inherit (umu-launcher-unwrapped) version meta;
 
-  # Use umu-launcher-unwrapped from the package args, to simplify overriding
-  targetPkgs = _: [ umu-launcher-unwrapped ];
+  # Use umu-launcher-unwrapped from the package args, to support overriding
+  targetPkgs = pkgs: [ pkgs.umu-launcher-unwrapped ] ++ extraPkgs pkgs;
```

Comment:
You've added the comment back (thanks), but not the implementation :grin: 

To be clear, you need to replace `pkgs.umu-launcher-unwrapped` with `umu-launcher-unwrapped`:

```suggestion
  targetPkgs = pkgs: [ umu-launcher-unwrapped ] ++ extraPkgs pkgs;
```

or


```suggestion
  targetPkgs = pkgs: extraPkgs pkgs ++ [
    umu-launcher-unwrapped
  ];
```

or 
```suggestion
  targetPkgs = pkgs: extraPkgs pkgs ++ [
    # Use umu-launcher-unwrapped from the package args, to support overriding
    umu-launcher-unwrapped
  ];
```

(I like the last one best, because it keeps the comment close to the relevant impl)